lib/scrabble-project/scrabble-class.rb

```diff
@@ -0,0 +1,90 @@
+module Scrabble
+
+  class Scrabble
+    # Set constants with general error message, point values of each letter, all letters
+    WORD_ERROR = "ERROR: Invalid input. Either your word has more that 7 letters or you entered something ridiculous. You only have 7 tiles, and Scrabble uses WORDS!"
+    POINTS = {
+      1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+      2 => ["d", "g"],
+      3 => ["b", "c", "m", "p"],
+      4 => ["f", "h", "v", "w", "y"],
+      5 => ["k"],
+      6 => ["j", "x"],
+      7 => ["q", "z"]
+    }
+
+    LETTERS = ("a".."z").to_a
+
+    def self.get_points(word)
+      # set initial value of score
+      score = 0
+
+      # convert to string, make letters lower case, & push letters to array
+      word_array = word.to_s.downcase.split("").push
```

Comment:
Couple things:
1) `word` should already be a string, so there's no need to call `.to_s` on it. If you wanted to ensure that the argument passed into `.get_points` is a string, you might consider putting it in your `.valid_input?` method.
2) Since `.split("")` already returns an array, you don't need `.push` at the end, here, especially if you're not providing an argument to push. In other words,  you're already assigning `word_array` to the return value of `word.downcase.split("")`.  On the other hand, if you wanted to start off with `word_array` as an empty array, how would you use `.push` to populate that array?
3) Also, there is another method on String that does exactly what `.split("")` does, and does not require an argument. Can you find it in the documentation?